src/bootstrap.loader.js

```diff
@@ -67,7 +67,7 @@ module.exports.pitch = function(source) {
 
   const loglevel = config.loglevel;
 
-  global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG;
+  global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG === true || process.env.DEBUG === "true" || process.env.DEBUG === "yes" || process.env.DEBUG === 1;
```

Comment:
I'd like to see this logic in a function, and downcase before checking, so TRUE passes